README.adoc

```diff
@@ -6,13 +6,48 @@ ____
 
 == Create a VM
 
-. launch my interactive menu
+. launch my interactive menu using the *_create_vm.sh_* script (located in the "scripts" folder)
 +
 [,bash]
 ----
 sh scripts/create_vm.sh
 ----
-. into the vm launch this
+ 
+. the interactive menu looks like this :
++
+[,bash]
+----
+main menu:
+1: setup the vm
+2: list vms
+3: exit
+4: start the vm
+5: generate info-vm.txt
+6: remove archlinux iso
+7: delete the vm
+----
+
+. select the first option to begin the setup of a new VM and follow the steps
+
+  you can skip each step if you know what you are doing
```

Comment:
AsciiDoc: [admonitions](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-types)

It's a builtin to mark text as note.

```suggestion
NOTE: you can skip each step if you know what you are doing
```